umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md

```diff
@@ -47,7 +88,9 @@ In the **Projects** section, you can get an overview of all the Projects that ha
 
 ![Project overview](images/Projects-v10.png)
 
-Additionally, you can view if any Projects have been disabled under your Organization.
+It is also possible to see the plan, project status, payment status, creation date, region and number of environments for each of your projects.
```

Comment:
as there are more options to see in this case, maybe the image above needs to be updated with the new view as well? 💡

Reply:
That is a good point!

I will update the image!

Reply:
The image has been updated :-)

Reply:
awesome! Then all is good now 😊